### DISCUSSION TITLE
Update do higer version is inaccessible

### DISCUSSION THREAD

**Discussion by gsulko80** [okd-project/okd#708] [Q&A] [answered]
**Describe the bug**
Hello, my cluster after update to version 4.6.0-0.okd-2021-02-14-205305 don't see any new update. What could be the reason ??

**Version**
UPI 4.6.0-0.okd-2021-02-14-205305

**How reproducible**

**Log bundle**

`oc get nodes`
`NAME                                 STATUS   ROLES           AGE    VERSION`
`ocmasterc1-dev01.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049`
`ocmasterc1-dev02.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049`
`ocmasterc1-dev51.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049`

`oc get clusterversion`
`NAME      VERSION                         AVAILABLE   PROGRESSING   SINCE   STATUS`
`version   4.6.0-0.okd-2021-02-14-205305   True        False         28d     Cluster version is 4.6.0-0.okd-2021-02-14-205305`

`oc get clusteroperators`
`NAME                                       VERSION                         AVAILABLE   PROGRESSING   DEGRADED   SINCE`
`authentication                             4.6.0-0.okd-2021-02-14-205305   True        False         False      43h`
`cloud-credential                           4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`cluster-autoscaler                         4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`config-operator                            4.6.0-0.okd-2021-02-14-205305   True        False         False      126d
console                                    4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`csi-snapshot-controller                    4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`dns                                        4.6.0-0.okd-2021-02-14-205305   True        False         False      68d`
`etcd                                       4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`image-registry                             4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`ingress                                    4.6.0-0.okd-2021-02-14-205305   True        False         False      3d14h`
`insights                                   4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`kube-apiserver                             4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`kube-controller-manager                    4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`kube-scheduler                             4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`kube-storage-version-migrator              4.6.0-0.okd-2021-02-14-205305   True        False         False      3d14h`
`machine-api                                4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`machine-approver                           4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`machine-config                             4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`marketplace                                4.6.0-0.okd-2021-02-14-205305   True        False         False      3d14h`
`monitoring                                 4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`network                                    4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`node-tuning                                4.6.0-0.okd-2021-02-14-205305   True        False         False      68d`
`openshift-apiserver                        4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`openshift-controller-manager               4.6.0-0.okd-2021-02-14-205305   True        False         False      6d20h`
`openshift-samples                          4.6.0-0.okd-2021-02-14-205305   True        False         False      68d`
`operator-lifecycle-manager                 4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`operator-lifecycle-manager-catalog         4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`operator-lifecycle-manager-packageserver   4.6.0-0.okd-2021-02-14-205305   True        False         False      3d13h`
`service-ca                                 4.6.0-0.okd-2021-02-14-205305   True        False         False      126d`
`storage                                    4.6.0-0.okd-2021-02-14-205305   True        False         False      68d`

`oc get nodes -o wide`
`NAME                                 STATUS   ROLES           AGE    VERSION                INTERNAL-IP      EXTERNAL-IP   OS-IMAGE `  `KERNEL-VERSION            CONTAINER-RUNTIME`
`ocmasterc1-dev01.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049   10.221.194.241   <none>`       `Fedora CoreOS 33.20210117.3.2   5.10.12-200.fc33.x86_64   cri-o://1.19.1`
`ocmasterc1-dev02.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049   10.221.194.242   <none> `       `Fedora CoreOS 33.20210117.3.2   5.10.12-200.fc33.x86_64   cri-o://1.19.1v`
`ocmasterc1-dev51.fortisbank.com.pl   Ready    master,worker   126d   v1.19.2+f173eb4-1049   10.221.194.243   <none>  `
`Fedora CoreOS 33.20210117.3.2   5.10.12-200.fc33.x86_64   cri-o://1.19.1`

`oc get clusterversion/version -o jsonpath={.spec.channel}`
`stable-4.6`

`oc adm upgrade
Cluster version is 4.6.0-0.okd-2021-02-14-205305
No updates available. You may force an upgrade to a specific release image, but doing so may not be supported and result in downtime or data loss.`

**Comment by vrutkovs** [comment, marked as answer]
>oc get clusterversion/version -o jsonpath={.spec.channel}
> stable-4.6

OKD has only one channel and its `stable-4`

**Comment by gsulko80** [comment]
Working. Thanks for help !!!
